packages/arb-upgrades/hardhat.config.ts

```diff
@@ -0,0 +1,5 @@
+// use same config from bridge package
+import * as config from 'arb-bridge-eth/hardhat.config'
+import 'hardhat-deploy-ethers'
```

Comment:
the hardhat config in`arb-bridge-eth` does `import 'hardhat-deploy'`
is there any reason we use this here but not there? I'm afraid of us running into incompatible dependency version clashes in the future